src/main/java/ch/njol/skript/log/LogEntry.java

```diff
@@ -128,8 +138,25 @@ protected void finalize() {
 	public String toString() {
 		if (node == null || level.intValue() < Level.WARNING.intValue())
 			return message;
+
 		Config c = node.getConfig();
-		return message + from + " (" + c.getFileName() + ", line " + node.getLine() + ": " + node.save().trim() + "')";
+
+		ArgsMessage details;
+		ArgsMessage lineInfo = WARNING_LINE_INFO;
+		if (level.intValue() == Level.WARNING.intValue())  // warnings
+			details = WARNING_DETAILS;
+		else if (level.intValue() == Level.SEVERE.intValue()) { // errors
+			details = ERROR_DETAILS;
+			lineInfo = ERROR_LINE_INFO;
+		}
+		else // anything else
+			details = OTHER_DETAILS;
+		
+		return
+			Utils.replaceEnglishChatStyles(lineInfo.toString(String.valueOf(node.getLine()), c.getFileName()).replaceAll("\\\\n", "\n")) +
+			Utils.replaceEnglishChatStyles(details.toString(message).replaceAll("\\\\n", "\n")) +
+			from +
+			Utils.replaceEnglishChatStyles(LINE_DETAILS.toString(node.save().trim()).replaceAll("\\\\n", "\n"));
```

Comment:
Would it be possible to only have to call the chat styles replacement method once here?

Reply:
Due to the need of use of `ArgsMessage#toString` I found no other way to accomplish this behavior other than this, if you have any suggestions I would like to hear :)

Reply:
nvm, I made it only 1 call